backend/app/controllers/spree/admin/addresses_controller.rb

```diff
@@ -0,0 +1,65 @@
+module Spree
+  module Admin
+    class AddressesController < Spree::Admin::BaseController
+      before_action :initial_user
+
+      def show
+        render 'addresses'
+      end
+
+      def update
+        new_shipping_address = Spree::Address.immutable_merge(@user.ship_address, user_params[:ship_address_attributes])
+        new_bill_address = Spree::Address.immutable_merge(@user.bill_address, user_params[:bill_address_attributes])
+
+        if (new_shipping_address.valid? && new_bill_address.valid?)
+          @user.bill_address = new_bill_address
+          @user.ship_address = new_shipping_address
```

Comment:
Did you check that it's actually updating the user and not only assigning it in memory?

Reply:
yes, new values will be saved for  user.  I tested. I reloaded the page and addresses have already been displayed with new values.

Reply:
@tvdeyen  sorry, I don't fully understand what you mean? 

> _Then update the one record (passed as id), to be inline with the rest of the resource controllers and be RESTful in those cases?_